internal/models/config.go

```diff
@@ -121,6 +122,10 @@ type AutostartDeployRequest struct {
 func (c *NodeConfiguration) Validate() bool {
 	c.Errors = make([]error, 0)
 
+	if c.AllowDuplicateWorkloads == nil {
```

Comment:
we can get rid of this check and not use a pointer. The allow duplicate workloads field shouldn't be required as it has reasonable defaults (False)